core/pkg/artifacts/manifest.go

```diff
@@ -55,9 +55,9 @@ func NewManifestFromProto(proto *service.ArtifactManifest) (Manifest, error) {
 		manifest.Contents = contents
 	}
 	for _, entry := range proto.Contents {
-		extra := map[string]interface{}{}
+		extra := map[string]any{}
```

Comment:
```suggestion
		extra := make(map[string]any, len(entry.Extra))
```
or just `make` up to you, tho you know the capacity ahead of time

Reply:
other than if it errors out..